Neos.ContentRepository.Core/Classes/Feature/ContentStreamCommandHandler.php

```diff
@@ -65,10 +66,8 @@ public function handle(CommandInterface $command, ContentRepository $contentRepo
      * @throws ContentStreamAlreadyExists
      */
     private function handleCreateContentStream(
-        CreateContentStream $command,
-        ContentRepository $contentRepository
+        CreateContentStream $command
     ): EventsToPublish {
-        $this->requireContentStreamToNotExistYet($command->contentStreamId, $contentRepository);
```

Comment:
why have all the two `requireContentStreamToNotExistYet` checks and the method itself been removed? (see also in `handleCloseContentStream`)

Reply:
Ah i see because you also removed `ContentStreamFinder::hasContentStream` ... but this does not make the image clearer to me?

Reply:
Well no, I removed it because it was no longer in use after removing the calls, and I removed the calls, because at this point we want to rather look at workspaces, BUT I guess this case here is valid 🤷 I'll add it back